typescript_utilities/src/processor.ts

```diff
@@ -65,6 +65,8 @@ interface HTMLProcessor {
 //
 // For `<meta name=declare-issxg-var>` elements, they are replaced with
 // `<script>window.isSXG=...</script>`, where `...` is true or false.
+
+// TODO: Use Rust `process_html.rs` to do this.
```

Comment:
Add a comment somewhere in this file that the logic is duplicated in process_html.rs, and any changes here need to be replicated over there.